assets/src/edit-story/components/checklist/toggle/index.js

```diff
@@ -25,19 +26,28 @@ import { Icons } from '@web-stories-wp/design-system';
  * Internal dependencies
  */
 import { ToggleButton } from '../../toggleButton';
+import { useCategoryCount } from '../countContext';
+import { useCheckpoint } from '../checkpointContext';
+import { ISSUE_TYPES, PPC_CHECKPOINT_STATE } from '../constants';
 
 const MainIcon = styled(Icons.Checkbox)`
   height: 32px;
   width: auto;
   display: block;
 `;
 
-function Toggle({
-  isOpen = false,
-  popupId = '',
-  onClick = () => {},
-  notificationCount = 0,
-}) {
+function Toggle({ isOpen = false, popupId = '', onClick = () => {} }) {
```

Comment:
💭 nit: could use `noop` so that this function doesn't get re-created on every render.